configs/Nanostack_ETH_config.json

```diff
@@ -0,0 +1,16 @@
+{
+    "config": {
+       "network-interface":{
+            "help": "options are LWIP_ETHERNET,WIFI,MESH_LOWPAN_ND,MESH_THREAD, NANOSTACK_ETHERNET",
+            "value": "LWIP_ETHERNET"
```

Comment:
It doesn't go to IPv6 directly either, does it need something else as well?
Nanostack doesn't support IPv4.